.gitignore

```diff
@@ -36,6 +36,9 @@ coverage.xml
 *.cover
 .hypothesis/
 .pytest_cache/
+test-config.json
+db.sqlite
+db.sqlite-journal
```

Comment:
I ran the unit-tests locally and I added these temporary test files to `.gitignore`. I would recommend refactoring to [tempfile](https://docs.python.org/3/library/tempfile.html) approach [as described](https://simpleit.rocks/python/test-files-creating-a-temporal-directory-in-python-unittests/) in future.